packages/engines/src/index.ts

```diff
@@ -1,32 +1,32 @@
-import path from 'path'
-import { enginesVersion } from '@prisma/engines-version'
-import { download } from '@prisma/fetch-engine'
+import { enginesVersion } from "@prisma/engines-version";
+import { download } from "@prisma/fetch-engine";
+import path from "path";
```

Comment:
Not sure why the double quotes and ; made it here.

Would be good to have the same setting as here https://github.com/prisma/prisma/blob/master/src/.prettierrc.yml

Reply:
Yeah prettier just needs to be setup. I Will add that in this PR